src/components/body/content/lessons/Lessons.tsx

```diff
@@ -1,28 +1,55 @@
 import "./index.scss";
+import { 
+  FC, useEffect, useState 
+} from "react";
 import { FormattedMessage } from "react-intl";
 import LessonCard from "../../../LessonCard";
-import React from "react";
-import items from "../../../../assets/lessonCard.json";
 
-const Lessons: React.FC = () => {
+const Lessons: FC = () => {
+  const [lessons, setLessons] = useState([]);
+  const getLessonsUrl = "https://lessoner-project-2w3h.onrender.com/lessons";
+
+  useEffect(() => {
+    return () => {
+      fetch(getLessonsUrl)
+        .then((response) => response.json())
+        .then((lessons) => setLessons(lessons.records))
+        .catch((error) => console.log(error));
+    };
+  }, []);
+  
   return (
     <div className="wrapper__lessons">
       <FormattedMessage id="app.lessons" />
       <div className="lessons">
-        {items.map((obj) => (
-          <LessonCard key={obj.id} 
-            title={obj.title} 
-            status={obj.status}
-            duration={obj.duration}
-            imagePreview={obj.imagePreview}
-            id={obj.id}
-            published={obj.published}
-            view={obj.view}
-            category={obj.category}
-            rating={obj.rating}
-            totalVotes={obj.totalVotes}
-          />))
-        }
+        {lessons.map(
+          (obj: {
+            id: number;
```

Comment:
You do not need to write here lesson type, you need to import this type. It is already exists

Reply:
done